tss-esapi/tests/context_tests/tpm_commands/object_commands_tests.rs

```diff
@@ -251,23 +249,26 @@ mod test_activate_credential {
             .unwrap()
             .key_handle;
 
-        let (_, key_name, _) = context.read_public(key_handle).unwrap();
+        let (_, key_name, _) = context
+            .read_public(key_handle)
+            .expect("Call to read_public failed");
 
         let cred = vec![1, 2, 3, 4, 5];
 
         let (credential_blob, secret) = context
             .execute_without_session(|ctx| {
                 ctx.make_credential(key_handle, cred.try_into().unwrap(), key_name)
             })
-            .unwrap();
+            .expect("Call to make_credential failed");
 
-        context.set_sessions((session_1, session_2, None));
+        context.set_sessions((Some(session_1), Some(session_2), None));
 
         let decrypted = context
             .activate_credential(key_handle, key_handle, credential_blob, secret)
-            .unwrap();
+            .expect("Call to active_credential failed");
 
-        let expected = Digest::try_from(vec![1, 2, 3, 4, 5]).unwrap();
+        let expected =
+            Digest::try_from(vec![1, 2, 3, 4, 5]).expect("Failed to create digest for expected");
```

Comment:
Not looking for you to change this, but I think the `expect` instead of `unwrap` here doesn't add much because we know it won't fail - I'd say the cases where failing is unlikely/impossible can stay `unwrap` in general